src/services/users.js

```diff
@@ -127,6 +165,8 @@ module.exports = class UserHelper {
 			competency: userDetails.competency,
 			designation: userDetails.designation,
 			language: userDetails.language,
+			organization_name: userDetails.organization.name,
+			image: userDetails.image,
```

Comment:
@rakeshSgr we can add this in column i am thinking, anyways in ELEVATE we have user image

Reply:
ok